WordPress/gutenberg
# Font Families and Faces: disable autosaves using empty class

## What?

- Revises `wp_font_family` and `wp_font_face` post type registration to use an empty class to disable autosave routes
- Adds tests that autosave routes are not registered

## Why?

Autosaves are not appropriate for fonts because they are settings rather than content. Having your font settings automatically saved while editing them could be unpleasantly surprising!

## How?

Autosaves are disabled by providing a valid, empty class that doesn't extend `WP_REST_Controller` when registering the post type.

## Testing Instructions

- Make a GET request to `wp/v2`
- See that the routes do not include autosave endpoints for font families or faces (e.g. `/wp/v2/font-families/(?P<id>[\d]+)/autosaves` is _not_ present)
